curvefs/conf/client.conf

```diff
@@ -103,6 +103,8 @@ fuseClient.maxDataSize=1024
 # default refresh data interval 30s
 fuseClient.refreshDataIntervalSec=30
 fuseClient.warmupThreadsNum=10
+# enable acl
+fuseClient.enableACL=true
```

Comment:
If disableXattr is true, we still can't use ACL

Reply:
The current logic is OK. ACL-related xattr is a special xattr and is not affected by the disablexattr switch.